packages/app/next.config.js

```diff
@@ -0,0 +1,6 @@
+/** @type {import('next').NextConfig} */
+const nextConfig = {
+  transpilePackages: ["@stackly/ui"],
```

Comment:
Not sure if we need the UI outside of app at this point.

The only potential use for the UI lib is if we build the landing page on other repo and want to use some components.

Reply:
Moved inside the app